users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -201,5 +282,55 @@ def get_masked():
     return x
 
 
-def get_specaug_funcs() -> list:
-    return [sort_filters_by_center_freq, _mask, random_mask, transform]
+def transform_with_variance(data, max_time_num, max_time, variance_factor, total_max_number_masks, network):
```

Comment:
It was not clear to me how a combination of both would look like so I separated them. Scheduling could be useful. It will require another schedule parameter in the config for the same reason I needed a new max mask parameter.

Reply:
Right, if you just mask single channels based on their frequency response's variance, the sorting has nothing to do with this. I think more detailed control over the schedule would be helpful, though.